use-api.md

```diff
@@ -0,0 +1,188 @@
+
+```
+https://api.rockarch.org/objects?online=true&start_date=1950
+```
+
+## Understanding the data that comes back 
+We provide [JSON](https://www.json.org/json-en.html)-formatted data.
+
+Endpoints are paginated. We show 50 records per page by default. Pagination can be controlled via the following query parameters:
+ - `limit` to set how many records each page will return
+ - `offset` to request a specific page of results.
+
+Example: get the **second** page of agent records consisting of **10** records:
+
+```
+GET https://api.rockarch.org/agents?limit=10&offset=10
+```
+
+## Endpoints and parameters
+
+ Using the available endpoints and their parameters, you can construct queries to get data back from the API. 
+
+See the full OpenAPI schema at [https://api.rockarch.org/schema](https://api.rockarch.org/schema) to see how our API is structured and understand the data.
+
+### Endpoints
+
+| Endpoint | Description |
+|------|------|
+|/agents|Returns a list of agents. Agents are people, organizations or families.|
+|/agents/{id}|Returns data about an individual agent.|
+|/collections|Returns a list of collections. Collections are intellectually significant groups of records.|
+|/collections/{id}|Returns data about an individual collection.|
+|/collections/{id}/ancestors|Returns the ancestors of a collection.|
+|/collections/{id}/children|Returns the children of a collection.|
+|/collections/{id}/minimap|Returns data about where search hits are located within a collection.|
+|/objects|Returns a list of objects. Objects are intellectually significant groups of records that do not have children.|
+|/objects/{id}|Returns data about an individual object.|
+|/objects/{id}/ancestors|Returns the ancestors of an object.|
+|/terms|Returns a list of terms. Terms are controlled values describing topics, geographic places or record formats.|
+|/terms/{id}|Returns data about an individual term.|
+|/search|Performs search queries across agents, collections, objects and terms.|
+|/search/{id}|Performs search queries across a specific agent, collection, object or term.
+|/schema/|Returns the OpenAPI schema for the RAC API.|
+
+### Parameters
+Use our [browseable API](https://api.rockarch.org) to see which parameters are available for which endpoints. For example, [https://api.rockarch.org/collections](https://api.rockarch.org/collections) lists the filter and sort fields, or parameters, that are available for that endpoint at the top of the webpage.
+
+| Parameter | Description | Example |
+|------|------|------|
+|limit|Number of results to return per page.|limit=50|
+|offset|Number of results to return per page.|offset=50|
+|id|Unique identifier. The id is used in an endpoint path to point to a specific collection, object, agents, etc.|/collections/2HnhFZfibK6SVVu86skz3k|
```

Comment:
Not sure if this should be in the parameter section?

Reply:
I think so? I see it listed in the schema as a parameter.